### PR TITLE
enh(api): add doc and api test for wep update api

### PR DESCRIPTION
**Pull request by kduret**
## Description

add openapi doc and api test for wep update api

**Fixes**MON-12296

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.04.x
- [ ] 21.10.x
- [ ] 22.04.x
- [x] 22.10.x (master)

### CODE REVIEW

File: config/packages/Centreon.yaml [hunk 1]
```diff
@@ -226,6 +226,7 @@ services:
 
     Core\Platform\Application\Repository\ReadUpdateRepositoryInterface:
         class: Core\Platform\Infrastructure\Repository\FsReadUpdateRepository
+        arguments: ['%centreon_install_path%']
```
Comment (adr-mo) [inline config/packages/Centreon.yaml]: I guess you could also use "named arguments" to make it clearer in the configuration file
```suggestion
        arguments:
            $installDir: '%centreon_install_path%'
```

**Review by adr-mo — APPROVED** [review]

**Review by jeremyjaouen — APPROVED** [review]

File: doc/API/v22.10/Administration/updates.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+patch:
+  tags:
+    - Platform
+  summary: "Update centreon web"
```
Comment (cg-tw) [inline doc/API/v22.10/Administration/updates.yaml]:
```suggestion
  summary: "Update Centreon web"
```

File: doc/API/v22.10/Administration/updates.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  description: |
+    Update centreon web component
```
Comment (cg-tw) [inline doc/API/v22.10/Administration/updates.yaml]:
```suggestion
    Update Centreon web component
```

**Review by adr-mo — APPROVED** [review]

File: tests/php/Core/Platform/Infrastructure/Repository/FsReadUpdateRepositoryTest.php [hunk 4]
```diff
@@ -32,20 +33,22 @@
     $this->finder = $this->createMock(Finder::class);
 });
 
-it('should not find updates if install directory does not exist', function () {
-    $repository = new FsReadUpdateRepository($this->filesystem, $this->finder);
+it('should returns an error when install directory does not exist', function () {
```
Comment (cg-tw) [inline tests/php/Core/Platform/Infrastructure/Repository/FsReadUpdateRepositoryTest.php]:
```suggestion
it('should return an error when install directory does not exist', function () {
```

**Review by adr-mo — APPROVED** [review]

**Review by jeremyjaouen — APPROVED** [review]

**Pull request merged**